src/content/docs/errors-inbox/errors-inbox.mdx

```diff
@@ -46,10 +46,12 @@ To enable errors inbox, follow these steps. Afterwards, errors groups should sta
 
 1. From [one.newrelic.com](https://one.newrelic.com) select **Errors inbox** from the top nav.
 2. If this is your first time accessing errors inbox, you will be prompted to select a [workload](/docs/new-relic-one/use-new-relic-one/workloads/workloads-isolate-resolve-incidents-faster/) in the top left.
-3. If you have no workloads set up, you will be prompted [to create one](/docs/new-relic-one/use-new-relic-one/workloads/use-workloads/) before you can use errors inbox. 
+3. If you have no workloads set up, you will be prompted [to create one](/docs/new-relic-one/use-new-relic-one/workloads/use-workloads/) before you can use errors inbox. Alternatively, you can  use [scoped views](#scoped) for browser and mobile monitoring without creating a workload.
```

Comment:
@bradleycamacho, what was this scoped services material doing at the end of the third-party services (before you chopped out the service info)?  Do scoped services somehow relate to these third-party services?